src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -174,64 +201,99 @@ properties:
         - Clonal Hematopoiesis
         - CNS Infection
         - Colon Polyps
+        - Colonization, Bacterial
+        - Colonization, Fungal
+        - Common Variable Immune Deficiency (CVID)
         - Common Variable Immunodeficiency
         - Congestive Heart Failure (CHF)
         - Connective Tissue Disorder
         - COPD
         - Coronary Artery Disease
+        - Cortisol Excess
+        - Cowden Syndrome
         - Crohn's Disease
         - Cryptococcal Meningitis
         - Cryptogenic Organizing Pneumonia
+        - Cyst(s)
         - Cytomegalovirus (CMV)
         - Deep Vein Thrombosis / Thromboembolism
         - Denys-Drash Syndrome
         - Depression
         - Dermatomyosis
         - Diabetes
+        - Diabetes, NOS
+        - Diabetes, Type I
         - Diabetes, Type II
         - Diabetic Neuropathy
         - Diet Controlled Diabetes
         - Diverticulitis
+        - Diverticulosis
         - Down Syndrome
         - DVT/PE
         - Dyslipidemia
         - EBV Lymphoproliferation
         - Eczema
+        - Endometriosis
+        - Endosalpingiosis
         - Epilepsy
+        - Epithelial Dysplasia
+        - Epithelial Hyperplasia
         - Epstein-Barr Virus
         - Escherichia coli
+        - Estrogen Excess
         - Familial Adenomatous Polyposis
         - Fanconi Anemia
         - Fibromyalgia
         - Fibrosis
         - Follicular Adenoma
+        - Gastric Polyp(s)
         - Gastritis
         - Gastroesophageal Reflux Disease
         - GERD
+        - Gilbert's Syndrome
         - Glaucoma
+        - Glomerular Disease
         - Glycogen Storage Disease
         - Goiter
         - Gonadal Dysfunction
         - Gorlin Syndrome
         - Gout
         - Graves' Disease
         - Hashimoto's Thyroiditis
+        - Hay Fever
         - Headache
         - Heart Disease
         - Helicobacter pylori Infection
+        - Helicobacter pylori-Associated Gastritis
```

Comment:
the capitalization seems a bit weird to me here but I've no idea what's right.

Reply:
This is a species name, so this is odd-looking but correct.